errbot/bootstrap.py

```diff
@@ -64,6 +64,8 @@ def bot_config_defaults(config):
         config.TEXT_DEMO_MODE = True
     if not hasattr(config, 'BOT_ADMINS'):
         raise ValueError('BOT_ADMINS missing from config.py.')
+    if not hasattr(config, 'BOT_ADMINS_NOTIFICATIONS'):
+        config.BOT_ADMINS_NOTIFICATIONS = ()
```

Comment:
I basically do that here:
https://github.com/sijis/err/blob/ffe23db0720d4cb90328e87896b2d5b437ada3c5/errbot/core.py#L565

it probably makes more sense to add it where you mention instead.